etna/models/autoarima.py

```diff
@@ -45,126 +41,11 @@ def __init__(
             Training parameters for auto_arima from pmdarima package.
         """
         self.kwargs = kwargs
-        self._model: Optional[ARIMA] = None
-        self.regressor_columns: List[str] = []
+        super().__init__()
```

Comment:
This should be before `self.kwargs` I guess.

Reply:
Why? In `class AutoARIMAModel(PerSegmentPredictionIntervalModel)` we set `self.kwargs` before calling super constructor.

Reply:
By mistake, no? 
It seems logic in parent class could reset self and so it's better to make `__init__` call before

Reply:
And vise-versa, logic in current class can reset the logic in the base class and broke smth.